packages/babel-core/src/api/browser.js

```diff
@@ -1,112 +0,0 @@
-/* eslint max-len: 0 */
```

Comment:
I've checked [package.json](https://github.com/babel/babel/blob/master/packages/babel-core/package.json) but I haven't seen any mention of [browser.js](https://github.com/babel/babel/blob/master/packages/babel-core/src/api/browser.js). Am I missing something or possibly looking in the wrong `package.json` ?